pkg/types/powervs/platform.go

```diff
@@ -14,6 +14,9 @@ type Platform struct {
 	UserID string `json:"userid"`
 	APIKey string `json:"apikey"`
 
+	DNSDomainName   string `json:"dnsdomainname"`
+	CISInstanceCRN  string `json:"cisinstancecrn"`
+	ResourceGroupID string `json:"resourcegroupid"`
```

Comment:
I see this in the ibmcloud code as well, but I don't actually see it ever used anywhere. Do you know why it's been added?